prow/plugins/bugzilla/bugzilla.go

```diff
@@ -819,6 +827,65 @@ func handleMerge(e event, gc githubClient, bc bugzilla.Client, options plugins.B
 	return comment(fmt.Sprintf("%s %s\n%s", mergedMessage("Some"), unmergedMessage, outcomeMessage("")))
 }
 
+func handleCherrypick(e event, gc githubClient, bc bugzilla.Client, options plugins.BugzillaBranchOptions, log *logrus.Entry) error {
+	comment := e.comment(gc)
+	// get the info for the PR being cherrypicked from
+	pr, err := gc.GetPullRequest(e.org, e.repo, e.cherrypickFromPRNum)
+	if err != nil {
+		log.WithError(err).Warn("Unexpected error getting title of pull request being cherrypicked from.")
+		return comment(formatError(fmt.Sprintf("checking the state of a cherrypicked pull request at https://github.com/%s/%s/pull/%d", e.org, e.repo, e.cherrypickFromPRNum), bc.Endpoint(), e.bugId, err))
```

Comment:
We should make it clear what we were doing when we ran into this issue, since the user took no specific action to ask the bot to do anything here. 

```
Failed to create a cherry-pick bug in Bugzilla: could not get ...
```